lib/web_ui/lib/src/engine/view_embedder/embedding_strategy/custom_element_embedding_strategy.dart

```diff
@@ -34,7 +34,10 @@ class CustomElementEmbeddingStrategy implements EmbeddingStrategy {
       ..style.height = '100%'
       ..style.display = 'block'
       ..style.overflow = 'hidden'
-      ..style.position = 'relative';
+      ..style.position = 'relative'
+      // This is needed so the browser lets flutter handle all pointer events
+      // it receives, without canceling them.
```

Comment:
Maybe also cross-link with the full page embedding?

Reply:
There's a slightly different explanation in the full-page embedding:

* https://github.com/flutter/engine/blob/main/lib/web_ui/lib/src/engine/view_embedder/embedding_strategy/full_page_embedding_strategy.dart#L57-L60

(Not sure which one I like the most :P)